internal/store/build_result.go

```diff
@@ -139,8 +140,8 @@ type K8sBuildResult struct {
 func (r K8sBuildResult) TargetID() model.TargetID   { return r.id }
 func (r K8sBuildResult) BuildType() model.BuildType { return model.BuildTypeK8s }
 
-// For kubernetes deploy targets.
-func NewK8sDeployResult(id model.TargetID, uids []types.UID, hashes []k8s.PodTemplateSpecHash, appliedEntities []k8s.K8sEntity) BuildResult {
+// NewK8sDeployResult creates a deploy result for Kubernetes deploy targets.
```

Comment:
ooh look at that properly formatted docstring!

Reply:
IntelliJ aggressively highlights them so I miiiight opportunistically fix them up if I'm touching the method anyway 😅